proposed/MPID Messaging.md

```diff
@@ -396,47 +384,221 @@ impl MpidMessage {
 }
 ```
 
-To send an MPID Message, a client would do something like:
-
-```rust
-let mpid_message = MpidMessage::new(my_mpid: Mpid, recipient: ::routing::Authority::Client,
-                                    metadata: Vec<u8>, body: Vec<u8>);
-
-```
-
 Account types held by MpidManagers
 
 ```rust
-struct Outbox {
+struct OutboxAccount {
     pub sender: ::routing::NameType,
+    pub sender_clients: Vec<::routing::Authority::Client>,
```

Comment:
We don't need to keep the authority (which contains the proxy ID for the Client's current session) do we?  Just the public signing key will do unless we want to start sending notifications back to the sender (i.e. messages which Vaults initiate to the Client as opposed to ones which are responses to Client requests).